v2/account_claims.go

```diff
@@ -72,26 +71,36 @@ func (j *JetStreamLimits) IsJSEnabled() bool {
 	return j.MemoryStorage != 0 || j.DiskStorage != 0
 }
```

Comment:
The fact that TieredLimits is not part of JetStreamLimits (although they are JS limits) as Alberto suggested, means that JetStreamLimits.ISJSEnabled() would return false here, even if there are tiered limits. Is that correct?

Reply:
I will push an update with refactored limits, so we can look at the alternative.

Reply:
Turns out IsJSEnabled was added with the previous (HAResources commit) and has not been released.